### PR TITLE
Documentation for Binary PDFs

### PR DESCRIPTION
**Pull request by averyallen**
Updates Binary resource documentation to reflect upcoming changes around support for reading documents with pdf content type.

### CODE REVIEW

**Comment by MadhurRaj** [comment]
+1

File: content/millennium/dstu2/infrastructure/binary.md [hunk 1]
```diff
@@ -29,20 +31,21 @@ List an individual Binary by its id:
...
-* This is usually linked from DiagnosticReport and should generally be accessed using the exact link given in that resource. Modifying the link has undefined consequences.
+* This is usually linked from DocumentReference or DiagnosticReport and should generally be accessed using the exact link given in the referring resource. Modifying the link has undefined consequences.
+* Documents containing URL sections are unsupported with contentType of `application/pdf`
 * See the [headers](#headers) section for concerns about the Accept header.
 * See the [authorization](#authorization-types) section for concerns about the required OAuth scopes.
 
 ### Authorization Types
 
-Requires both the appropriate Binary.read and DiagnosticReport.read scopes to be granted to the caller.
+The necessary OAuth scopes can be determined by checking the attachment.contentType in the referring resource. For attachments with the `application/pdf` contentType, the Binary.read and DocumentReference.read scopes are required. For attachments with the `text/html` contentType, the Binary.read and DiagnosticReport.read scopes are required.
```
Comment (MadhurRaj) [inline content/millennium/dstu2/infrastructure/binary.md]: The scopes might need to change now.

Comment (averyallen) [reply content/millennium/dstu2/infrastructure/binary.md]: The scopes can't be updated until after a local service change coming out at the end of the month.

**Comment by davischristi** [comment]
Looks good. 👍

**Pull request merged**